target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -98,6 +98,13 @@ netgear,ex6150v2)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:router" "phy0tpt"
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:client" "phy1tpt"
 	;;
+netgear,ex6200v2)
+        ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:wlan2g" "phy0radio"
+        ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:wlan5g" "phy1radio"
+        ucidef_set_led_usbport "usb" "USB" "green:usb" "usbport" "usbdev" "usb4" "usb3" "usb2" "usb1" "usb3-port1" "usb2-port1" "usb1-port1"
+        ucidef_set_led_netdev "lan" "LAN" "green:router" "phy0assoc" "phy1assoc"        
```

Comment:
What is this LED used for?

Cause it doesn't make sense to have it trigger if something is associated with the WLAN radios

Reply:
Per the OEM manual this LED is used to show when the extender is connected to a device. However, I've since used this LED for power_amber and power_green so this is a carryover, will remove. 

![image](https://user-images.githubusercontent.com/86842761/207955141-fc08c319-be28-42ef-9036-49db187ca10b.png)